cmd/puppeth/wizard_genesis.go

```diff
@@ -240,8 +240,8 @@ func (w *wizard) manageGenesis() {
 		w.conf.Genesis.Config.SetEIP145Transition(w.readDefaultUint64P(*w.conf.Genesis.Config.GetEIP145Transition()))
 
 		fmt.Println()
-		fmt.Printf("Which block should YOLOv1 (EIP2537) come into effect? (default = %v)\n", w.conf.Genesis.Config.GetEIP2537Transition())
-		w.conf.Genesis.Config.SetEIP2537Transition(w.readDefaultUint64P(*w.conf.Genesis.Config.GetEIP2537Transition()))
+		fmt.Printf("Which block should YOLOv2 come into effect? (default = %v)\n", w.conf.Genesis.Config.GetEIP2929Transition())
```

Comment:
@meowsbits there is not really an EIP for YoloV2 yet, so I used the EIP2929 as the flag for that

Reply:
Yeap, here's the EIP-as-fork-indicator scenario again. This might eventually warrant it's own logic somewhere (a fork :: indicator EIP function). Another important place where this happens is here, for example: https://github.com/etclabscore/core-geth/blob/master/core/vm/evmc.go#L302

Reply:
Thanks for sharing the evmc link. Good to know that something like this exists.